web/docs/database/backends.md

```diff
@@ -1,3 +1,329 @@
 ---
-title: Database Backends
+title: Databases
 ---
+
+[Entities](/docs/database/entities.md), [Operations](/docs/database/operations.md) and [Automatic CRUD](/docs/database/crud.md) together make a high-level interface for working with your app's data. Still, all that data has to live somewhere, so let's see how Wasp deals with databases.
+
+## Supported Database Backends
+
+Wasp supports multiple database backends. We'll list and explain each one.
+
+### SQLite
+
+The default database Wasp uses is [`SQLite`](https://www.sqlite.org/index.html).
+
+SQLite is a great way for getting started with a new project because it doesn't require any configuration, but Wasp can only use it in development. Once you want to deploy your Wasp app to production, you'll need to switch to `PostgreSQL` and stick with it.
+
+Fortunately, migrating from SQLite to PostgreSQL is pretty simple, and we have [a guide](#migrating-from-sqlite-to-postgresql) to help you.
+
+### PostgreSQL
+
+[PostgreSQL](https://www.postgresql.org/) is a real production database.
```

Comment:
This reads poorly, we should somehow say "this is rock solid battle tested database" without implying that SQLite is not (since it has proven itself recently) .

Something I found online:
> PostgreSQL is the world’s most advanced open source database and the fourth most popular database. In development for more than 20 years, PostgreSQL is managed by a well-organized and highly principled and experienced open source community. It is an object-oriented database that is fully ACID compliant and highly extensible, enabling the community to add new features and capabilities as workload demands evolved.

But maybe more relaxed 🤷